### PR TITLE
Feat/component loader

### PR DESCRIPTION
**Pull request by jwasiak**

### CODE REVIEW

File: src/log.action.ts [hunk 1]
```diff
@@ -53,8 +53,8 @@ const getRecordTitle = (modifiedRecord, currentAdmin) => {
 export const createLogAction =
   ({
     onlyForPostMethod = false,
-    options = {},
-  }: CreateLogActionParams = {}): After<ActionResponse> =>
+    options,
```
Comment (dziraf) [inline src/log.action.ts:56]: Why did you remove defaults here and below? 🤔

File: src/types.ts [hunk 2]
```diff
@@ -83,6 +84,7 @@ export type LoggerFeatureOptions = {
    * In case you want to use different attribute names, you can use this
    * options to configure the mapping.
    */
+  componentLoader?: ComponentLoader;
```
Comment (dziraf) [inline src/types.ts:87]: Why is `componentLoader` optional? It should also be on different line because comment above is related to `propertiesMapping` prop.

File: src/log.action.ts (hunk 1 above)
Comment (jwasiak) [reply src/log.action.ts]: reverted

File: src/types.ts (hunk 2 above)
Comment (jwasiak) [reply src/types.ts]: fixed
